pallets/nfts/src/lib.rs

```diff
@@ -19,6 +19,13 @@ use sp_runtime::{DispatchResult, RuntimeDebug};
 use sp_std::result;
 use ternoa_common::traits::{LockableNFTs, NFTs};
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Default, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct SeriesDetails {
+    pub series_id: u128,
+    pub item_id: u64,
```

Comment:
`item_id` in series? Why is that?

Reply:
I thought that an individual distinguished could be made if every nft has it's own id. But this feature might not be needed.